__tests__/account.test.ts

```diff
@@ -106,86 +105,6 @@ describe('deploy and test Wallet', () => {
     // this is tested indirectly true declareAndDeploy while declaring
   });
 
-  describeIfDevnetSequencer('Test on Devnet Sequencer', () => {
```

Comment:
I think these tests can stay, it's just that they would not be wrapped anymore inside the `describeIfDevnetSequencer` 
cc @tabaktoni

Reply:
Yes, but instead of sequencer it should be now:
describeIfRPCGoerli.
Because this \ test can run only on Goerli network